src/tools/post_processor/post_processor/main.cc

```diff
@@ -48,6 +52,13 @@ ReturnCode Patch(MachOFile *, const PatchSettings &);
 
 }  // namespace
 
+bool nextToken(std::istringstream &str, std::string &tok, char delim) {
+  if (!std::getline(str, tok, delim)) {
+    fprintf(stderr, "Invalid format: use sed-style ,needle,new_needle,");
```

Comment:
another place where the rest of the code uses `stderr`, so it's probably best to maintain the same style as the rest of the code